Server/htdocs/AppController/commands_RSM/api/v2/items/getCount.php

```diff
@@ -0,0 +1,105 @@
+<?php
+// ****************************************************************************************
+//Description:
+//    Returns the number of items from the specified itemType with filter conditions
+//
+//  --- PARAMETERS -- All the IDs can be called as systemNames --
+//   RStoken      : A token can replace the clientID
```

Comment:
The clientID can never replace a token. A token is mandatory always when working with the API